src/main/resources/iudx/catalogue/server/validator/resourceItemSchema.json

```diff
@@ -1,65 +1,78 @@
 {
```

Comment:
I think the diffs loaded are formatting only except for max length properties? Can you confirm?

Reply:
yeah, maxLength:512